AppUseCaseDiagram.drawio

```diff
@@ -1,152 +1,202 @@
+                        <mxPoint x="670" y="680" as="sourcePoint"/>
+                        <mxPoint x="790" y="720.5" as="targetPoint"/>
                         <mxPoint as="offset"/>
                     </mxGeometry>
                 </mxCell>
-                <mxCell id="6" value="&amp;lt;&amp;lt;extend&amp;gt;&amp;gt;" style="html=1;verticalAlign=bottom;labelBackgroundColor=none;endArrow=open;endFill=0;dashed=1;fillColor=#1ba1e2;strokeColor=#3399ff;jumpSize=6;strokeWidth=2;entryX=1;entryY=0.5;entryDx=0;entryDy=0;exitX=0.033;exitY=0.36;exitDx=0;exitDy=0;exitPerimeter=0;" edge="1" parent="1" source="ugMZ8tAmapSa5t5XAWcm-9" target="ugMZ8tAmapSa5t5XAWcm-8">
+                <mxCell id="6" value="&amp;lt;&amp;lt;extend&amp;gt;&amp;gt;" style="html=1;verticalAlign=bottom;labelBackgroundColor=none;endArrow=open;endFill=0;dashed=1;fillColor=#1ba1e2;strokeColor=#3399ff;jumpSize=6;strokeWidth=2;entryX=1;entryY=0.5;entryDx=0;entryDy=0;exitX=0.033;exitY=0.36;exitDx=0;exitDy=0;exitPerimeter=0;" parent="1" source="ugMZ8tAmapSa5t5XAWcm-9" target="ugMZ8tAmapSa5t5XAWcm-8" edge="1">
                     <mxGeometry x="0.0013" width="160" relative="1" as="geometry">
-                        <mxPoint x="420" y="459.5" as="sourcePoint"/>
-                        <mxPoint x="540" y="500" as="targetPoint"/>
+                        <mxPoint x="400" y="619.5" as="sourcePoint"/>
+                        <mxPoint x="520" y="660" as="targetPoint"/>
                         <mxPoint as="offset"/>
                     </mxGeometry>
                 </mxCell>
-                <mxCell id="7" value="&amp;lt;&amp;lt;extend&amp;gt;&amp;gt;" style="html=1;verticalAlign=bottom;labelBackgroundColor=none;endArrow=open;endFill=0;dashed=1;fillColor=#1ba1e2;strokeColor=#3399ff;jumpSize=6;strokeWidth=2;entryX=0.5;entryY=1;entryDx=0;entryDy=0;exitX=0.5;exitY=0;exitDx=0;exitDy=0;" edge="1" parent="1" source="ugMZ8tAmapSa5t5XAWcm-8" target="ugMZ8tAmapSa5t5XAWcm-7">
+                <mxCell id="7" value="&amp;lt;&amp;lt;extend&amp;gt;&amp;gt;" style="html=1;verticalAlign=bottom;labelBackgroundColor=none;endArrow=open;endFill=0;dashed=1;fillColor=#1ba1e2;strokeColor=#3399ff;jumpSize=6;strokeWidth=2;entryX=0.5;entryY=1;entryDx=0;entryDy=0;exitX=0.5;exitY=0;exitDx=0;exitDy=0;" parent="1" source="ugMZ8tAmapSa5t5XAWcm-8" target="ugMZ8tAmapSa5t5XAWcm-7" edge="1">
                     <mxGeometry x="-0.1429" width="160" relative="1" as="geometry">
-                        <mxPoint x="370" y="360" as="sourcePoint"/>
-                        <mxPoint x="490" y="400.5" as="targetPoint"/>
+                        <mxPoint x="350" y="520" as="sourcePoint"/>
+                        <mxPoint x="470" y="560.5" as="targetPoint"/>
                         <mxPoint as="offset"/>
                     </mxGeometry>
                 </mxCell>
-                <mxCell id="8" value="Login to Service Provder Interface" style="ellipse;whiteSpace=wrap;html=1;fillColor=#66B2FF;" vertex="1" parent="1">
```

Comment:
Suggest changing line 123 to read "Login to Service Provider Interface:

Reply:
fixed